index.js

```diff
@@ -0,0 +1,68 @@
+const addon = require('bindings')('glue');
+
+const pkcs7_header = Buffer.from('-----BEGIN PKCS7-----');
+const pkcs7_footer = Buffer.from('-----END PKCS7-----');
+const nl = Buffer.from('\n');
+
+/**
+ * Verify a document given a public key, document and a PKCS#7 encoded
+ * signature.  All parameters should either be strings, Buffers or something
+ * which can be converted into a Buffer safely with a call to Buffer.from(),
+ * with an encoding parameter of 'utf-8'.
+ */
```

Comment:
nit: comment doesn't mention what the function returns or errors it can throw; consider linking to the #API or #Errors section of the readme

Reply:
Good idea.  I've included a text reference, but when its landed, I'll insert links.